### DISCUSSION TITLE
我应该什么时候交作业/答辩呢

### DISCUSSION THREAD

**Discussion by yokooseed** [west2-online/learn-go#4] [Q&A] [answered]
RT

**Comment by ozline** [comment, marked as answer]
每年的考核通知、结果都以群为主, 所以如果你还没加群, 请尽快加群


关于交作业和答辩:

1. 交作业每年的时间都不一样, 参考加群后各个方向的通知, 这个仓库只是为了存放作业要求和相关材料
2. 答辩也是如此, 不过难度很低, 相当于体验一下答辩的感觉, 没有很难啦

综上所述, 如果你想查阅更多关于时间的内容, 你应该**加群以获得每年的要求**

另外, 我们全年纳新, 你什么时候进群都可以哦
